filebeat/magefile.go

```diff
@@ -196,12 +196,14 @@ func IntegTest() {
 // GoIntegTest starts the docker containers and executes the Go integration tests.
 func GoIntegTest(ctx context.Context) error {
 	mg.Deps(BuildSystemTestBinary)
+	mg.Deps(Build)
```

Comment:
Why is this necessary? That you need both the regular and system test binary seems strange because they are still both filebeat.

Reply:
I see later on its because of the benchmark test

Reply:
I'll mark this as resolved and we can discuss the use of the real binary in the other comment.